tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,72 @@
+## **Create New Identity With Attributes Using Local Fabric CA**
+`5-10 mins`
+
+> **Important:** You should have a **smart contract** instantiated on your Local Fabric to follow this tutorial. Follow part 1 of this series first for instructions.
+
+
+## Learning Objectives
+
+* Create a new identity (with attributes) using Local Fabric CA.
+* Using the newly created identity and implement attribute-based action control.
+
+---
+<details>
+<summary><b>1. Creating a new identity with attributes</b></summary>
+
+In the very first tutorial (Tutorial 1: Local smart contract development), we used a pre-set identity (`admin`) to connect to the Local Fabric gateway and submit/evaluate transactions. In this tutorial, we will learn how to create _our own_ identity with attributes.
```

Comment:
I don't think you want to repeat yourself here, i think you should refer to the tutorial about creating an identity here and then talk about why you would want to create an identity with attributes

Reply:
Fixed